pgml-dashboard/src/components/confirm_modal/template.html

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Why this extra whitespace?

Reply:
Hmm, trim should of worked, good question.